observ/proxy.py

```diff
@@ -16,9 +19,9 @@ class Proxy:
     """
 
     __hash__ = None
-    __slots__ = ["target", "readonly", "shallow", "__weakref__"]
+    __slots__ = ("target", "readonly", "shallow", "__weakref__")
```

Comment:
There's a few more places where we define `__slots__` with `[]`, you want to update those as well?

Reply:
Ah no, that was not intended...
Passing a list to `__slots__` is useless since slots are supposed to be immutable, but I guess this change slipped through the formatter... Do you want me to revert this line?

Reply:
I like the change you made - switching to tuples! Please keep it in and apply it to all instances of `__slots__` :)